posts/2022-01-04-azure-eventing.md

```diff
@@ -0,0 +1,496 @@
+
+For example, consider Organization A that adopts an event-driven microservices architecture for core systems and Organization B that still relies on error-prone batch processes.  The value of Organization A's investment is limited as soon as it attempts to collaborate with Organization B.  Whether the interaction among those organizations takes the form of periodic SFTP transfers (Organization B's preference) or point-to-point API interactions (Organization A's preference), neither duplicates the real-time, event-driven interactions from a shared source of truth that Organization A aspired to create in-house.
+
+As [partner ecosystems become central](https://www.bearingpointbeyond.com/en/insights/market-insights/partner-ecosystem/) to unlocking new business opportunities, it's imperative that Organizations, no matter their real-time data sharing maturity, have an easy path toward joint real-time, event-driven solutions.  Further, it's equally important to allow each Organization to make the most of its substantial investment in its in-house real-time data sharing capabilities.  For example, collaboration between Organization A and Organization B should be able to leverage the substantial investment Organization A made to modernize its in-house systems.  More often than not, this means not only working seamlessly across organizational boundaries but also seamlessly across Cloud Service Providers (CSPs), as it's common for partners to adopt different CSPs while still wanting to work effectively together.
+
+Vendia Share addresses those imperatives through the concept of a Universal Application (or Uni for short).  A Uni allows each party (what we call a Uni participant) to work from a single source of truth, removing the complexity of sharing data among parties while also streamlining the use of the data as it's shared.  "As it's shared" means applying real-time actions to real-time data, using an event-driven programming model.  It also means working across organizations no matter their choice and investment in differing CSPs.
+
+In this blog post we'll highlight the value of Vendia Share's new Azure capabilities from both a cross-cloud data sharing and event-driven programming perspective.  Consider the case of a Supplier (on AWS), working through a Distributor (on AWS), who delivers goods to a Retailer (on Azure).  When the Supplier makes an adjustment (say by changing the anticipated fulfilment date of an existing purchase order), both the Distributor and the Retailer would like to be made aware (and take action) immediately.
+
+<figure>
+  <img src="https://user-images.githubusercontent.com/85032783/147981825-f0e94c34-6488-4fd3-82f4-7ef8e00224a2.png" width="100%"/>
+  <figcaption align="center"><b>Figure 1</b><br/><i>A Supplier, Distributor, and Retailer reacting in real-time across clouds as changes to purchase orders occur</i></figcaption>
```

Comment:
```suggestion
  <figcaption align="center"><b>Figure 1</b> - <i>A Supplier, Distributor, and Retailer reacting in real-time across clouds as changes to purchase orders occur</i></figcaption>
```

Reply:
Just saves a little real estate on the page. Not sure how it will look. Feel free to keep what you have.

Reply:
Let's find out :).